doc/content/end-user-documentation/settings.rst

```diff
@@ -224,3 +224,21 @@ Testing
 .. setting::
     :name: TEST_INPUT_EVENT_TIMEOUT
     :path: lona.default_settings.TEST_INPUT_EVENT_TIMEOUT
+
+Server
+------
+
+.. setting::
+    :name: AIOHTTP_CLIENT_MAX_SIZE
+    :path: lona.default_settings.AIOHTTP_CLIENT_MAX_SIZE
+
+    .. note::
```

Comment:
Can you please remove this message? The next version could be 1.11 as well.

Reply:
I hope I have understood correctly. I have now  removed the ``..note::``section.
I have updated my branch accordingly.